File: Arithmetic App/simple_arithmetic_app.py
import random

def get_random_number():
    return random.randint(1, 100)


count = 0
score = 0

while count < 10:
    first_number = get_random_number()
    second_number = get_random_number()

    attempts = 0
    result = -1
    user_response = 0

    while attempts < 2 and user_response != result:
        if first_number > second_number:
            result = first_number - second_number
            user_response = int(input(f"What is {first_number} - {second_number}? "))
        else:
            result = second_number - first_number
            user_response = int(input(f"What is {second_number} - {first_number}? "))

        if user_response == result:
            print("Correct!!!")
            score += 1
        else:
            print("Wrong. Try again")

        attempts += 1

    print()
    count += 1

print(f"You got {score} out of 10.")

File: Arithmetic App/SimpleArithmeticApp.java
import java.util.Random;
import java.util.Scanner;

public class SimpleArithmeticApp {
    public static void main(String... args) {
        Random randomGenerator = new Random();
        Scanner inputCollector = new Scanner(System.in);

        int count = 0;
        int score = 0;

        do {
            int firstNumber = randomGenerator.nextInt(100);
            int secondNumber = randomGenerator.nextInt(100);

            int result = 0;
            int userResponse = 0;
            int attemptCount = 0;

            while(attemptCount < 2) {
                if(firstNumber > secondNumber) {
                    result = firstNumber - secondNumber;
                    System.out.printf("What is %d - %d? ", firstNumber, secondNumber);
                } else {
                    result = secondNumber - firstNumber;
                    System.out.printf("What is %d - %d? ", secondNumber, firstNumber);
                }
                userResponse = inputCollector.nextInt();

                if(result == userResponse) {
                    System.out.println("Correct!!!");
                    score++;
                    break;
                } else {
                    System.out.println("Wrong. Try again!");
                }

                attemptCount++;
            }

            System.out.println();

            count++;
        } while(count < 10);

        System.out.printf("You got %d out of 10.%n", score);
    }
}